workers/unity/Assets/Playground/Scripts/Camera/FollowCameraSystem.cs

```diff
@@ -22,6 +22,7 @@ namespace Playground
     ///     ScrollWheel: controls the orbit radius, moving the camera closer to or
     ///     farther from the player.
     /// </remarks>
+    [DisableAutoCreation]
```

Comment:
we might not want all systems to have these. It' more about letting the user decide whether to add a system automatically to the default world or not.

Though I agree that we might not want to do this. This commit is simply in here to show we could get rid of our reflection stuff. If people, prefer to do that instead of using the way that Unity currently proposes (which is, to be fair, quite cumbersome) then I'm fine with that as well ;)

Reply:
Hopefully these can go away when unity gives better support for multiple worlds in a game.